docs/docs/path_algorithm.md

```diff
@@ -0,0 +1,406 @@
+---
+title: "Path algorithms"
```

Comment:
Similar comment as above for `docs/docs/images/graph_query_road.png`

Reply:
Same.